### ISSUE TITLE
Non-namespaced global symbols: Travis should fail even more...

### ISSUE THREAD

**Issue by tlepoint** [open-quantum-safe/liboqs#130]
In addition to #128, current travis builds that are flagged as passing output what is below, and therefore should fail.

```
Code contains the following non-namespaced global symbols; see https://github.com/open-quantum-safe/liboqs/wiki/Coding-conventions for function naming conventions.
00000000000000c0 T allocateProof
0000000000000060 T allocateRandomTape
0000000000000130 T allocateSignature
0000000000000190 T allocateUnruhPermutations
0000000000000000 T allocateView
00000000000003b0 T cleanup_EVP
0000000000000520 T Commit
0000000000002450 T computeRoundKey
0000000000001460 T computeRoundKey2
0000000000000f50 T deserializeSignature
0000000000000110 T freeProof
00000000000000a0 T freeRandomTape
0000000000000160 T freeSignature
00000000000001c0 T freeUnruhPermutations
0000000000000040 T freeView
0000000000000660 T G
00000000000005c0 T generateAesIv
00000000000002b0 T getAllRandomness
0000000000000230 T getBit
00000000000001e0 T getBitFromWordArray
00000000000008b0 T getChallenge
0000000000000280 T getRandom32
0000000000000950 T H3
0000000000000390 T init_EVP
0000000000002320 T mpc_AND
00000000000011c0 T mpc_AND_verify
0000000000002660 T mpc_LowMC
0000000000002250 T mpc_multiplyWithKeyMatrix
0000000000001390 T mpc_multiplyWithKeyMatrix2
0000000000002130 T mpc_multiplyWithMatrix
0000000000001280 T mpc_multiplyWithMatrix2
0000000000002460 T mpc_substitution
0000000000001470 T mpc_substitution2
00000000000003e0 T mpc_xor
00000000000004e0 T mpc_xor2
0000000000000480 T mpc_xor4_2
0000000000000420 T mpc_xor4_3
0000000000000290 T numBytes
00000000000008f0 T print_hex
0000000000000bd0 T prove
0000000000002890 T random_bytes_default_openssl
0000000000002850 T runMPC
0000000000000d30 T serializeSignature
0000000000000250 T setBit
0000000000000200 T setBitInWordArray
0000000000000850 T setChallenge
00000000000028b0 T sign
0000000000001aa0 T verify
0000000000001680 T verifyLowMC
0000000000001840 T verifyProof
0000000000000620 T freeLookupTables
0000000000000610 T freeRoundConstants
0000000000000650 T LowMCEnc
0000000000000230 T readLookupTables
0000000000000130 T readRoundConstants
0000000000000000 T get_transform
0000000000000040 T picnic_get_param_name
00000000000001e0 T picnic_init
0000000000000290 T picnic_keygen
0000000000000060 T picnicParamsToLowMCParams
0000000000000b70 T picnic_read_private_key
0000000000000a90 T picnic_read_public_key
0000000000000250 T picnic_shutdown
0000000000000530 T picnic_sign
0000000000000760 T picnic_signature_size
0000000000000bf0 T picnic_validate_keypair
0000000000000830 T picnic_verify
0000000000000b00 T picnic_write_private_key
0000000000000a20 T picnic_write_public_key
```

**Comment by dstebila** [comment]
Shravan @smashra and I already talked about this; he will work on it.

**Comment by dstebila** [comment]
Fixed by #154
